microsoftgraph/microsoft-graph-explorer-v2
# Feat/show auth in progress

## Overview

Shows a spinner with the message `Signing you in...`  while the user is logging in.

### Demo

![image](https://user-images.githubusercontent.com/12011447/68658856-4a902900-0547-11ea-9c0b-1f73f70fbc2c.png)

### Notes

N/A

## Testing Instructions
* Launch Graph Explorer
* Click on Sign In
* Observe the spinner with the respective text

Closes #260
